bin/setup

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
```

Comment:
1. Whats this file about?
2. How do you use this file?

Reply:
It allows you to write commands that will run to setup your project. A developer should be able to clone the project and run a single, consistent, reliable command, i.e. `setup` to start the project.

Reply:
how do you run this command / file?

Reply:
If you're in the root directory, you can run `./bin/setup`.